### DISCUSSION TITLE
histoire ui breaks when adding pinia store

### DISCUSSION THREAD

**Discussion by KeremCevik** [histoire-dev/histoire#468] [Q&A]
hello guys,
i am currently working on histoire integration in my project.
I am encountering the following problem:
As soon as I import a piniastore in a story or in the setupfile and use its useSomeStore, the histoire ui breaks. And it looks like this:
![image](https://user-images.githubusercontent.com/126464761/222721552-aad8c39e-9d6c-46e3-8b46-4c6abf90dbed.png)

with kind regards
Kerem

**Comment by jowilhnson** [comment]
Will probably require more information for anyone to be able to help you. What does your setup file look like? How are you trying to import and use pinia in your story?

Myself, I have had no problems, but I have simply setup histoire to use pinia so that my vue components that are using the store will work correctly in histoire. I am not calling on the store in my story files themselves. 

My setup file looks like this:

```
import { createPinia } from 'pinia'
import { defineSetupVue3 } from '@histoire/plugin-vue'

export const setupVue3 = defineSetupVue3(({ app, story, variant }) => {
	app.use(createPinia())
})
```
And have that file set as the setupFile in histoire.config. That was enough to get components using pinia to display in histoire.
